corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -112,15 +112,25 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
             <div>
                 <!-- ko if: help() -->
                 <a
-                    class="help-popover"
-                    tabindex="0"
+                    class="help-text-trigger"
                     role="button"
-                    data-trigger="focus"
                     data-placement="top"
-                    data-toggle="popover"
-                    data-bind="attr: { 'data-content': help }">
+                    data-toggle="modal"
+                    data-target="#help-text-modal">
                     <i class="fa fa-info-circle"></i>
                 </a>
+                <div id="help-text-modal" class="modal fade" role="dialog" tabindex='-1'>
```

Comment:
Does this definitely work if there are multiple questions with help text on the same page? The semantics for `id` are that it's a unique identifier, so if you try to `getElementById` (which bootstrap is presumably doing somewhere) and there are two of that id on the page, I don't know that it'll get the one you want.

Might be better to add a click handler that calls `.modal('show')` rather than using the `data-*` attributes to control these modals.